doc/input-output-reference/src/overview/group-performance-tables.tex

```diff
@@ -1,819 +1,629 @@
+This group of objects consists of tabular data which are used to characterize the performance of HVAC equipment. The use of performance tables eliminate the need to perform a regression analysis to calculate the performance curve equation coefficients. The following descriptions define the input requirements for the performance table objects.
 
-\subsection{Table:OneIndependentVariable}\label{tableoneindependentvariable}
+\subsection{Table:Lookup}\label{tablelookup}
 
-Input for the data table consists of a table name, the curve type, the interpolation method, and the data pairs representing the polynomial equation. Optional inputs for minimum and maximum values for the independent variable and curve minimum and maximum may be used to limit the output of the table when evaluating the regression curve (e.g., limit extrapolation). The existing performance curves represented by this table object are:
+Input for tables representing data with one or more independent
+variables.
 
-\begin{equation}
-Linear\,\,Curve:\,\,\,Output = {C_1} + {C_2}x
-\end{equation}
+\subsubsection{Inputs}\label{inputs}
 
-\begin{equation}
-Quadratic\,\,Curve:\,\,\,Output = {C_1} + {C_2}x + {C_3}{x^2}
-\end{equation}
+\paragraph{Field: Name}\label{field-name}
 
-\begin{equation}
-Cubic\,\,Curve:\,\,\,Output = {C_1} + {C_2}x + {C_3}{x^2} + {C_4}{x^3}
-\end{equation}
+A unique user-assigned name for an instance of a lookup table. When a
+table is used, it is referenced by this name. The name of this table
+object may be used anywhere a valid performance curve object is allowed.
 
-\begin{equation}
-Quartic\,\,Curve:\,\,\,Output = {C_1} + {C_2}x + {C_3}{x^2} + {C_4}{x^3} + {C_5}{x^4}
-\end{equation}
+\paragraph{Field: Independent Variable List
+Name}\label{field-independent-variable-list-name}
 
-\begin{equation}
-Exponent\,\,Curve:\,\,\,Output = {C_1} + {C_2}{x^{{C_3}}}
-\end{equation}
+The name of a Table:IndependentVariableList object that defines the
+independent variables that comprise the dimensions of the tabular data.
 
-\subsubsection{Inputs}\label{inputs-032}
+\paragraph{Field: Normalize Output}\label{field-normalize-output}
 
-\paragraph{Field: Name}\label{field-name-031}
+Determines whether or not this output will be normalized, either through:
 
-A unique user assigned name for an instance of a performance table. When a table is used, it is referenced by this name. The name of this table object may be used anywhere a valid performance curve object is allowed.
+\begin{enumerate}
+  \item automatic normalization to 1.0 at the normalized reference conditions defined by each
+  independent variable in the associated independent variable list, or
+  \item use of the Normalization Value field, below.
+\end{enumerate}
 
-\paragraph{Field: Curve Type}\label{field-curve-type}
+If both methods are defined, the automatic normalization will take precedence.
```

Comment:
I'm not following this. Seems like the choice is Yes=Normalize, No=Don't normalize. The next field Normalization Value has a default of 1.0, so it's not clear how the input would look if you want the "automatic normalization", and it's not clear what the input would look like such that "both methods are defined".  Seems like blank should be significant here to trigger automatic normalization? So, it shouldn't have a default value?

Reply:
Yeah, this gets a little complicated. If Normalize Output = "No", no normalization will happen. If it is "Yes", and you have normalization references set for each of the independent variables, then those are used to normalize the output to 1.0 at the corresponding combination of independent variable values. If they are not set, then the output is simply normalized by the value set (or defaulted) by the normalization value set in the following field. Let me know if you want to have a quick chat to discuss. I'm open to changing this behavior.

Reply:
Changing the choices to:

- *No*: Do not normalize output.
- *DivisorOnly*: Normalize the output using only the value entered into the Normalization Divisor field, below.
- *AutomaticWithDivisor*: Normalize the output by first dividing the value of the table at each of the independent variable normalization reference values in the associated independent variable list, and then dividing by the value entered into the Normalization Divisor field, below.